### PR TITLE
Add jax.numpy.nanmin Frontend

### PR DESCRIPTION
**Pull request by mohamedehab0999**

### CODE REVIEW

**Comment by mohamedehab0999** [comment]
Closes #9384.
Tracking TODO issue #8423.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by vaatsalya123** [comment]
lgtm!
